### PR TITLE
[webapi][xwalk-3561] Add spec_maturity section to spec.json

### PR DESCRIPTION
**Pull request by kangxu**

### CODE REVIEW

File: webapi/tct-3dtransforms-css3-tests/3dtransforms/spec.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
  "3dtransforms": {
   "spec_url": "http://www.w3.org/TR/2012/WD-css3-transforms-20120911/",
+  "spec_maturity": "CR",
```
Comment (zqzhang) [inline webapi/tct-3dtransforms-css3-tests/3dtransforms/spec.json]: WD

File: webapi/tct-audio-html5-tests/audio/spec.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
  "audio": {
   "spec_url": "http://www.w3.org/TR/2012/WD-html5-20121025/the-audio-element.html#the-audio-element",
+  "spec_maturity": "REC",
```
Comment (zqzhang) [inline webapi/tct-audio-html5-tests/audio/spec.json]: This version shall be WD; the latest is REC.

File: webapi/tct-batterystatus-w3c-tests/batterystatus/spec.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
  "batterystatus": {
   "spec_url": "https://dvcs.w3.org/hg/dap/raw-file/default/battery/Overview.html",
+  "spec_maturity": "REC",
```
Comment (zqzhang) [inline webapi/tct-batterystatus-w3c-tests/batterystatus/spec.json]: CR

File: webapi/tct-browserstate-html5-tests/browserstate/spec.json [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {
  "browserstate": {
   "spec_url": "http://www.w3.org/TR/2012/WD-html5-20120329/offline.html?path=5_4_5#event-online",
+  "spec_maturity": "ED",
```
Comment (zqzhang) [inline webapi/tct-browserstate-html5-tests/browserstate/spec.json]: WD

File: webapi/tct-canvas-html5-tests/canvas-py/spec.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
   "canvas":{
     "spec_url":"http://www.w3.org/TR/2012/WD-2dcontext-20120329/",
+    "spec_maturity": "CR",
```
Comment (zqzhang) [inline webapi/tct-canvas-html5-tests/canvas-py/spec.json]: This version is WD; the latest is CR.

Can you please check the spec link carefully?? Please double check them all.

**Comment by kangxu** [comment]
@zhiqiang
I add spec_maturity section following the attachment in [xwalk-3561](https://crosswalk-project.org/jira/browse/XWALK-3561). What shall I do next,  update spec_maturity section following the spec_url section or update spec_url?

**Comment by jennycao** [comment]
merge this PR first for test suite release report .
Need to update the spec_URL in test suite: tests.full.xml, spec.json to align witht the crosswalk implementation.

**Pull request merged**